sklearn/decomposition/_pca.py

```diff
@@ -413,6 +413,10 @@ def __init__(
     def n_features_(self):
         return self.n_features_in_
 
+    @property
+    def loadings(self):
```

Comment:
Please add "_" in the end of function name:
"loadings" -> "loadings_"

Reply:
Complete!